array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(x => x * 2);
+console.log(doubled);
+
+const prices = numbers.map(x => '$' + x.toString() + '.00');
+console.log(prices);
```

Comment:
This should produce valid price strings for numbers that have their own decimal places